xrmocap/human_perception/bbox_detection/mmdet_trt_detector.py

```diff
@@ -0,0 +1,242 @@
+import cv2
+import logging
+import numpy as np
+from tqdm import tqdm
+from typing import Union
+from xrprimer.utils.ffmpeg_utils import video_to_array
+from xrprimer.utils.log_utils import get_logger
+
+from xrmocap.transform.bbox import qsort_bbox_list
+
+try:
+    from mmdeploy.apis.utils import build_task_processor
+    from mmdeploy.utils import get_input_shape, load_config
+    has_mmdeploy = True
+    import_exception = ''
+except (ImportError, ModuleNotFoundError):
+    has_mmdeploy = False
+    import traceback
+    stack_str = ''
+    for line in traceback.format_stack():
+        if 'frozen' not in line:
+            stack_str += line + '\n'
+    import_exception = traceback.format_exc() + '\n'
+    import_exception = stack_str + import_exception
+
+
+class MMdettrtDetector:
```

Comment:
```suggestion
class MMdetTrtDetector:
```
or
```suggestion
class MMdeployTrtDetector:
```
I prefer the second.